Workshop-Preparation.md

```diff
@@ -82,10 +86,22 @@ $ git lfs install
 
 `git clone https://github.com/Byont-Ventures/Hackathon.git`
 
-**Note:** We use [VSCode](https://code.visualstudio.com/) for development. There are other free-to-use tools, but we can only provide limited support if you decide to use those.
-
 If you open this repo in VSCode, you should get a suggestion to install a set of extensions; we recommend you do so.
 
+## 4.5 Configuration for Windows users
+
+Please note that if you are a Windows user, you will have to put in a lot more effort configuration-wise, and we will only be able to provide limited support.
+In short, you can use WSL to run the challenges and open them in VSC.
+Nevertheless, we will give you some general instructions on how to get things up and running:
+
```

Comment:
If you are a windows user, we recommend using Windows Subsystem for Linux (WSL) to work with Foundry.
The support is better, and configuration is (way) easier.
You can use the steps below to get started